error: 'ARDUINO_ESP8266_GIT_VER' was not declared in this scope

The latest master has build problems with `ARDUINO_ESP8266_GIT_VER`

```
WARNING: Category 'Network' in library lwIP_PPP is not valid. Setting to 'Uncategorized'
WARNING: Category 'Network' in library lwIP_enc28j60 is not valid. Setting to 'Uncategorized'
WARNING: Category 'Network' in library lwIP_w5500 is not valid. Setting to 'Uncategorized'
WARNING: Category 'Network' in library lwIP_w5500 is not valid. Setting to 'Uncategorized'
/home/runner/arduino_ide/hardware/esp8266com/esp8266/cores/esp8266/core_esp8266_main.cpp:76:78: error: 'ARDUINO_ESP8266_GIT_VER' was not declared in this scope; did you mean 'ARDUINO_ESP8266_GIT_DESC'?
   76 | extern const uint32_t __attribute__((section(".ver_number"))) core_version = ARDUINO_ESP8266_GIT_VER;
      |                                                                              ^~~~~~~~~~~~~~~~~~~~~~~
      |                                                                              ARDUINO_ESP8266_GIT_DESC
exit status 1
Error compiling for board Generic ESP8266 Module.
```

the full log can be found in the here:
https://github.com/Links2004/arduinoWebSockets/runs/2851579051?check_suite_focus=true

the code change that broke the master build needs to be in the last 7 days since before the the pipeline where running fin.
https://github.com/Links2004/arduinoWebSockets/actions

the core installation steps are here:
https://github.com/Links2004/arduinoWebSockets/blob/c897f605679d41aeb4f09325ab8070ad09d4cc15/travis/common.sh#L96-L103

if more info or tests are needed feel free to ask.

--- a/tools/makecorever.py
+++ b/tools/makecorever.py
@@ -31,10 +31,12 @@ def generate(path, platform_path, version="unspecified", release = False):
 
     text = ""
 
+    git_ver = "00000000"
     try:
-        text = "#define ARDUINO_ESP8266_GIT_VER   0x{}\n".format(git("rev-parse", "--short=8", "HEAD"))
+        git_ver = git("rev-parse", "--short=8", "HEAD")
     except Exception:
         pass
+    text = "#define ARDUINO_ESP8266_GIT_VER   0x{}\n".format(git_ver)
 
     # version is
     # - using Arduino-CLI:

